rxbinding-kotlin/src/main/kotlin/com/jakewharton/rxbinding/view/RxView.kt

```diff
@@ -109,6 +110,17 @@ public inline fun View.dragEvents(): Observable<ViewDragEvent> = RxView.dragEven
 public inline fun View.dragEvents(handled: Func1<in ViewDragEvent, Boolean>): Observable<ViewDragEvent> = RxView.dragEvents(this, handled)
 
 /**
+ * Create an observable of [ViewTreeObserverDrawEvent] for draws on `view`.
+ * 
+ * *Warning:* The created observable keeps a strong reference to `view`. Unsubscribe
+ * to free this reference.
+ * 
+ * *Warning:* The created observable uses [ViewTreeObserver.addOnDrawListener] to observe
+ * draws. Only one observable can be used for a view at a time.
```

Comment:
ViewTreeObserver allows more than one OnDrawListener to be registered therefore this doc line should be changed according to https://github.com/JakeWharton/RxBinding/issues/75.